tests/AheadOfTime/Trimming/check.ps1

```diff
@@ -43,7 +43,7 @@ function CheckTrim($root, $tfm, $outputfile, $expected_len) {
 # error NETSDK1124: Trimming assemblies requires .NET Core 3.0 or higher.
 
 # Check net7.0 trimmed assemblies
-CheckTrim -root "SelfContained_Trimming_Test" -tfm "net9.0" -outputfile "FSharp.Core.dll" -expected_len 286208
+CheckTrim -root "SelfContained_Trimming_Test" -tfm "net9.0" -outputfile "FSharp.Core.dll" -expected_len 287232
```

Comment:
A note to have a look at this.
cc: @KevinRansom

Reply:
@vitek-karas Do you happen to know if anything changed in trimmer, which could've caused that?

Reply:
Intentional might be this: https://github.com/dotnet/runtime/pull/105714
Depends on the exact setup of the app (if it enables debug support or not). It's a difference of 1KB so not that big.

It might also be unintentional, but that's hard to tell without investigation where the size increase comes from - and for such a small size change that is pretty tricky to do.

Reply:
@sbomer FYI

Reply:
@vitek-karas is there something we can do to help figuring it out? Will having dll from p6 trimmer and p7 help? We can probably do IL diff.

Reply:
Yes having a dll from previous version should help. If you have the old/new dlls, can you share it with us please ?